docs/Getting-Started.md

```diff
@@ -123,7 +123,7 @@ async function routes (fastify, options) {
   fastify.get('/', async (request, reply) => {
     return { hello: 'world' }
   })
-  
+<br>
```

Comment:
This one shouldn't be here. You should change your regex to something like `sed -i '' 's/(.+)(\s{2})$/\1<br>/g'`